tests/packet_test_client.toit

```diff
@@ -45,7 +53,7 @@ with_packet_client create_transport/Lambda [block]
     idle.down
 
   task::
-    exception := catch --unwind=(on_handle_error == null):
+    exception := catch --unwind=(on_handle_error == null) --trace:
```

Comment:
Do you want to keep the tracing turned on?

Reply:
Removed it. It's just in the test, but I don't think we need it.